html/pfappserver/lib/pfappserver/Form/Violation.pm

```diff
@@ -186,6 +186,13 @@ has_field 'redirect_url' =>
    tags => { after_element => \&help,
              help => 'Destination URL where PacketFence will forward the device. By default it will use the Redirection URL from the portal profile configuration.' }
   );
+has_field 'external_command' =>
```

Comment:
Should we be deprecating the old way of using external commands with this new way ?